content/docs/hooks-faq.md

```diff
@@ -303,17 +305,17 @@ function useWindowPosition() {
 }
 ```
 
-Note how we were able to move the `useState` call for the `position` state variable and the related effect into a custom Hook without changing their code. If all state was in a single object, extracting it would be more difficult.
+Nota cómo podemos mover el llamado a `useState` para la variable de estado `position` y el effecto relacionado en un Hook personalizado sin cambiar su código. Si todo el estado estuviera en un solo objeto, extraerlo sería más difícil.
 
-Both putting all state in a single `useState` call, and having a `useState` call per each field can work. Components tend to be most readable when you find a balance between these two extremes, and group related state into a few independent state variables. If the state logic becomes complex, we recommend [managing it with a reducer](/docs/hooks-reference.html#usereducer) or a custom Hook.
+Ambas aproximaciones, poner todo el estado en un solo llamdo a `useState`, y usar un llamado a `useState` por cada campo, pueden funcionar. Los Componentes suelen ser más legibles cuando encuentras un balance entre ambos extremos, y agrupas partes del estado relacionadas en unas cuantas variables de estado independientes. Si la lógica del estado se vuelve muy compleja, recomendamos [manejarla con un reductor](/docs/hooks-reference.html#usereducer), o un Hook personalizado.
 
-### Can I run an effect only on updates? {#can-i-run-an-effect-only-on-updates}
+### ¿Puedo correr un efecto solo en cuando hayan actualizaciones? {#can-i-run-an-effect-only-on-updates}
```

Comment:
I suggest `¿Puedo correr un efecto solo cuando hayan actualizaciones?`

Reply:
Yes this was definitley something I missed, thanks!